subsys/bluetooth/services/hids.c

```diff
@@ -879,6 +880,9 @@ int bt_hids_init(struct bt_hids *hids_obj,
 {
 	LOG_DBG("Initializing HIDS.");
 
+	__ASSERT(init_param->rep_map.size <= BT_ATT_MAX_ATTRIBUTE_LEN,
```

Comment:
This is an API function. I think that assert could be to restrictive here. Please consider returning error code here

Reply:
I have changed it to an error code now